README.md

```diff
@@ -9,15 +9,27 @@ wherever it is possible. Unfortunately legacy builds `coreboot-4.0.x` require
 toolchain from pre-coreboot-sdk era that's why we created
 `pce-fw-builder-legacy`.
 
+Keep in mind that this tool works only for coreboot releases not older than
+v4.6.9 and v4.0.17. For older releases use the procedure described in
+[release_manifests](https://github.com/pcengines/release_manifests).
+Also, due to a typo in configuration file, when building v4.6.9 you will be asked
+if you want to `Include CPU microcode in CBFS` - just press Enter and the build
+process will continue.
+
 Usage
 -----
 
 Initial run may take some time. Below procedures assume that Docker is
 correctly installed and current user is in `docker` group. Script automatically
 detect with which codebase it deals with and choose toolchain accordingly.
 
+If you don't want to use default containers take a look at [this paragraph](#building-docker-image).
+
+```
+$ git clone https://github.com/pcengines/pce-fw-builder.git -b v1.2.0
```

Comment:
@krystian-hebel giving exact version makes no sens. I understand this is just an example, but if anybody would copy paste, it can cause problems. Second thing, we would have to change that line every release. Maybe better use `-b <most_recent_tag>`?